nucypher/cli/commands/deploy.py

```diff
@@ -461,14 +462,14 @@ def contracts(general_config, actor_options, mode, activate, gas, ignore_deploye
     deployment_parameters = {}
     if parameters:
         with open(parameters) as json_file:
-            deployment_parameters = json.load(json_file)
+            deployment_parameters = json.load(json_file)  # TODO: Seems like this is bypassing existing flow in economics.py
```

Comment:
This is meant to be used by contracts that have non-default constructor parameters, like the MultiSig (currently not used).

Reply:
Ah - I see.